src/Administration/Notification/NotificationDefinition.php

```diff
@@ -17,32 +17,44 @@
 use Shopware\Core\Framework\DataAbstractionLayer\Field\ManyToOneAssociationField;
 use Shopware\Core\Framework\DataAbstractionLayer\Field\StringField;
 use Shopware\Core\Framework\DataAbstractionLayer\FieldCollection;
+use Shopware\Core\Framework\Feature;
 use Shopware\Core\Framework\Log\Package;
 use Shopware\Core\System\Integration\IntegrationDefinition;
 use Shopware\Core\System\User\UserDefinition;
 
+/**
+ * @deprecated tag:v6.8.0 - Will be removed in 6.8.0. Use Shopware\Core\Framework\Notification\NotificationDefinition instead
+ */
 #[Package('framework')]
 class NotificationDefinition extends EntityDefinition
```

Comment:
While this doesn't extend the core notification definition, technically if you type hinted on this class as a result from say the definition registry